x-pack/plugins/maps/public/connected_components/toolbar_overlay/set_view_control/decimal_degrees_form.tsx

```diff
@@ -90,7 +90,6 @@ export class DecimalDegreesForm extends Component<Props, State> {
             onChange={this._onLatChange}
             isInvalid={isLatInvalid}
             data-test-subj="latitudeInput"
-            step="any" // Browsers will validate decimals as invalid otherwise
```

Comment:
For context for this reversion, EuiFieldNumber now defaults to `step="any"`, so all decimal number inputs will no longer have invalid states (although you're still of course free to pass in your own custom step interval, e.g. `step={0.5}`